src/core/agent/services/credentialService.ts

```diff
@@ -604,6 +604,177 @@ class CredentialService extends AgentService {
       }
     }
   }
+
+  private async waitForEnterpriseAcdcToAppear(
+    serverAid: string,
+    expectedDomain: string,
+    retryTimes: number,
+    triedTime = 0
+  ) {
+    const notificationsList =
+      await this.agent.modules.signify.getNotifications();
+    let unreadGrantNotes = notificationsList.notes.filter(
+      (note: any) => !note.r && note.a.r === "/tunnel/server/request"
```

Comment:
Here we are waiting for an enterprise grant - so it's `/exn/ipex/grant` not `/tunnel/server/request`